File: chat-service/src/main/java/com/yazikochesalna/chatservice/config/properties/FrontProperties.java
package com.yazikochesalna.chatservice.config.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;
import org.springframework.stereotype.Component;

import java.util.List;

@Component
@ConfigurationProperties(prefix = "front.server")
@Data
public class FrontProperties {

    private List<String> origins;
}

File: authorization-service/src/main/kotlin/com/yazikochesalna/authorizationservice/config/properties/FrontProperties.kt
package com.yazikochesalna.authorizationservice.config.properties

import lombok.Data
import org.springframework.boot.context.properties.ConfigurationProperties
import org.springframework.stereotype.Component


@Component
@ConfigurationProperties(prefix = "front.server")
@Data
class FrontProperties {
    lateinit var origins: List<String>
}
